### PR TITLE
Include move_amount.py

### PR DESCRIPTION
**Pull request by OdatNurd**
Include the move_amount command that was originally written on the
Sublime forum and just recently cross posted to Stack Overflow. This
implements a drop in replacement for the move command that allows you
to specify how many times the action should happen.

### CODE REVIEW

File: plugins/README.md [hunk 1]
```diff
@@ -43,6 +43,11 @@ of each file for more information on full usage.
    on a window-by-window basis (not view-by-view), so some chicanery is needed
    to pull this off.
 
+ * [move_amount.py](move_amount.py) is a simple drop in replacement for the
+   standard `move` command that allows you to specify the number of times the
+   movement happens. This can be handy for certain kinda of navigation or in
```
Comment (keith-hall) [inline plugins/README.md]: I think there's a typo here - `kinda of` should be `kinds of`

File: plugins/move_amount.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import sublime
+import sublime_plugin
+
+# Related reading:
+#     https://forum.sublimetext.com/t/command-to-move-by-several-lines/28508
+#     https://forum.sublimetext.com/t/command-to-move-by-several-lines/28508
```
Comment (keith-hall) [inline plugins/move_amount.py]: related reading links are the same, should there be two different links or just the one?

**Review by keith-hall — CHANGES_REQUESTED** [review]
couple of small documentation/comment related improvements

**Comment by OdatNurd** [comment]
Typo and link fixed. I also enabled a project setting to ensure newlines at the ends of files. We may want to run a change to re-save all files to get them into a good state.

**Review by keith-hall — APPROVED** [review]

**Pull request merged**